ansible_collections/arista/avd/molecule/eos_designs-twodc-5stage-clos/intended/structured_configs/DC1-POD1-L2LEAF2A.yml

```diff
@@ -31,9 +31,7 @@ management_api_http:
   enable_vrfs:
     MGMT: {}
   enable_https: true
-eos_cli: "interface Loopback1002\n  description Loopback created from raw_eos_cli\
-  \ under l2leaf node-group RACK2_MLAG\n\ninterface Loopback1111\n  description Loopback\
-  \ created from raw_eos_cli under platform_settings vEOS-LAB\n"
+eos_cli: "interface Loopback1002\n  description Loopback created from raw_eos_cli under l2leaf node-group RACK2_MLAG\n\ninterface Loopback1111\n  description Loopback created from raw_eos_cli under platform_settings vEOS-LAB\n"
```

Comment:
Would it be ok to use literal block scalars for such commands as below:

eos_cli: |
         interface Loopback1002
           description Loopback created from raw_eos_cli under l2leaf node-group RACK2_MLAG
         
         interface Loopback1111
           description Loopback created from raw_eos_cli under platform_settings vEOS-LAB

Reply:
We do use block scalars in the input data, but in the structured_config this is converted into a regular string. We tried many different approaches and this was the most robust one.